docs/btcli-permissions.md

```diff
@@ -120,36 +120,38 @@ See: [Coldkey and Hotkey Workstation Security](./getting-started/coldkey-hotkey-
 
 ### `wallet`
 
-Wallet subcommands have a variety of uses and must be run on all different kinds of workstation.
+`wallet` subcommands have a variety of uses and must be run on all different kinds of workstation.
 
-This command is required to provision keys to `btcli`, so it can access your wallet. This is essentially the equivalent of logging in/authentication. This is true for both coldkeys (which all users require), and hotkeys (required only by miners and validators, and for advanced functions).
+The `walleet` command is required to provision keys to `btcli`, so btcli can access your wallet. This is essentially the equivalent of logging in/authentication. This is true for both coldkeys, which all users require, and hotkeys, which are required only by miners and validators as well as for advanced functions.
```

Comment:
```suggestion
The `wallet` command is required to provision keys to `btcli`, so it can access your wallet. This is essentially the equivalent of logging in/authentication. This is true for both coldkeys, which all users require, and hotkeys, which are required only by miners and validators as well as for advanced functions.
```